docs/_news/news11.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: news
+title: New for Codewind 0.11.0
+description: New for Codewind 0.11.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news11
+---
+
+## Codewind 0.11.0
+Thursday, 16 April 2020
+
+You might have noticed we skipped 0.10.0, but it's because we are committed to making the best tooling available to you. Our last development cycle just needed a little more love. 💕
+
+#### ✨ New Features and Highlights for 0.11.0 ✨
+**IntelliJ Tech Preview Updates**
+
+We've continued to work hard on getting the IntelliJ experience up and consistent with our other IDEs. Here are the latest IntelliJ updates:
+
+- [IntelliJ supports **Restart in Debug Mode**](https://github.com/eclipse/codewind/issues/990)
+- [View project log files in IntelliJ](https://github.com/eclipse/codewind/issues/989).
+- Open and [view the performance dashboard](https://github.com/eclipse/codewind/issues/2038) for select project types.
+- [Enable and disable projects](https://github.com/eclipse/codewind/issues/987).
+- [Simultaneously select and remove multiple projects](https://github.com/eclipse/codewind/issues/2225).
+- Create a new project with the [**New Project** menu action](https://github.com/eclipse/codewind-intellij/pull/64).
+
+![video of new feature, creating a new project with IntelliJ](images/imagesfornews/newprojintellij.gif){:width="800"}
+
+**Codewind Website**
+- We are pleased to announce that our blogs now have a [home](https://www.eclipse.org/codewind/blog.html) on our website! 🏡
+
+**IDEs**
+##### VS Code, Eclipse, and IntelliJ
+- [Appsody projects no longer show build status](https://github.com/eclipse/codewind/issues/2052). 
+
+##### VS Code and Eclipse Che
+- The Codewind extension downloads and activates significantly faster. [The extension size is reduced by 98% to 1.6 MB](https://github.com/eclipse/codewind/issues/1060). 
+- You only need to download the `cwctl` and `Appsody` binaries one time. They are fetched from the internet when the [extension is activated for the first time](https://github.com/eclipse/codewind-vscode/pull/506). 
```

Comment:
To me, `You only...` implies the user has to actively download the binaries, which is not true.  It might be better to say something like `The cwctl and appsody binaries are automatically downloaded from the internet when the extension is activated for the first time`